### ISSUE TITLE
Not available to install via pip

### ISSUE THREAD

**Issue by chuckhendo** [agronholm/mqttproto#26]
The install instructions (https://mqttproto.readthedocs.io/en/latest/userguide.html#installing) say to run `pip install mqttproto` - however, it doesn't seem to be available.

**Comment by agronholm** [comment]
Yeah, because it's not ready for publishing yet. It will be on the PyPI when that changes.

**Comment by liaoya** [comment]
Please update the document. I hope it will be on PyPI soon.
